dev_env/api/rootfs/app-entrypoint.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+INIT_SEM=/tmp/initialized.sem
+PACKAGE_FILE=glide.lock
+
+log () {
+  echo -e "\033[0;33m$(date "+%H:%M:%S")\033[0;37m ==> $1."
+}
+
+dependencies_up_to_date() {
+  # It it up to date if the package file is older than
+  # the last time the container was initialized
+  [ ! $PACKAGE_FILE -nt $INIT_SEM ]
+}
+
+if [ "$1" == "go" -a "$2" == "run" ]; then
+	if ! dependencies_up_to_date; then
+		log "Packages updating..."
+		glide install
```

Comment:
It will install the new dependencies if glide.lock has changed on container restart.